src/test/java/org/kiwiproject/dropwizard/error/ErrorContextBuilderTest.java

```diff
@@ -130,6 +146,53 @@ private void assertIllegalArgumentExceptionThrownBuilding(SoftAssertions softly,
         }
     }
 
+    @Nested
+    class CleanupJob {
+
+        @Test
+        void shouldHaveCleanupApplicationErrorsJob() {
+            var cleanupConfig = new CleanupConfig();
+            cleanupConfig.setCleanupJobName("foo-%d");
+
+            var builder = ErrorContextBuilder.newInstance()
+                    .environment(environment)
+                    .serviceDetails(serviceDetails)
+                    .cleanupConfig(cleanupConfig)
+                    .dataStoreType(DataStoreType.NOT_SHARED);
+
+            builder.buildInMemoryH2();
+
+            var dataSourceFactory = ApplicationErrorJdbc.createInMemoryH2Database();
+            builder.buildWithDataStoreFactory(dataSourceFactory);
+
+            var jdbi = Jdbi3Builders.buildManagedJdbi(environment, dataSourceFactory);
+             builder.buildWithJdbi3(jdbi);
```

Comment:
Format this line (it's indented one space too far). And yes, it _really_ matters or else the code won't work. :laughing: